build/recipes.mk

```diff
@@ -3,7 +3,7 @@
 # Makefile fragment for recipe actions
 #
 EMBEDDED_RECIPES_PATH         ?= internal/install/recipes/files
-RECIPE_ARCHIVE_VERSION        ?= $(shell curl -Ls https://download.newrelic.com/install/open-install-library/currentVersion.txt)
+RECIPE_ARCHIVE_VERSION        ?= $(shell curl -Ls https://s3.us-east-1.amazonaws.com/nr-downloads-main/install/open-install-library/currentVersion.txt)
 RECIPE_ARCHIVE_URL            ?= https://download.newrelic.com/install/open-install-library/${RECIPE_ARCHIVE_VERSION}/recipes.zip
```

Comment:
@ctrombley don't we need to also update RECIPE_ARCHIVE_URL

Reply:
👍